Nome: Estevão Calebe Almeidas Vilas Boas

Matricula: 97276

# Probabilidade e Estatística - 1ª Lista de Exercícios - 2023

1ª Questão: Considere a distribuição de 300 estudantes segundo sexo e área de concentração em uma escola como na tabela 1. Um estudante é escolhido ao acaso. Determine:

   a) a probabilidade de que seja do sexo feminino e da área de humanas
    
   b) a probabilidade de que seja do sexo masculino e não seja da área de biológicas
    
   c) a probabilidade de que seja do sexo feminino dado que é da área de humanas

Resolução:

a)
$$ P(F \cap H) = P(F \mid H)\cdot P(H) = \frac{80}{138} \cdot \frac{138}{300} = \frac{4}{15} \approx 26.66\% $$

b) 
$$ P(M \cap B^c) = P(M \mid B^c)\cdot P(B^c) = \frac{98}{210}\cdot \frac{210}{300} = \frac{98}{300} \approx 32.66\% $$

c)
$$ P(F) = \frac{80}{138} \approx 57.97\% $$


In [1]:
from fractions import Fraction

# a)
P_F_H = (Fraction(80,138) * Fraction(138,300))
print(f'a) {P_F_H} ou {float(P_F_H):.2%}')

# b)
P_M_Bc = (Fraction(98,210) * Fraction(210,300))
print(f'b) {P_M_Bc} ou {float(P_M_Bc):.2%}')

# c)
P_F = (Fraction(80,138))
print(f'c) {P_F} ou {float(P_F):.2%}')

a) 4/15 ou 26.67%
b) 49/150 ou 32.67%
c) 40/69 ou 57.97%


---

2ª Questão: Um aluno responde a um teste de múltipla escolha com 4 alternativas e apenas 1 correta. A probabilidade de que ele saiba a resposta correta de uma questão é de 30%. Se ele não sabe existe a possibilidade de acertar no chute (não existe a possibilidade de cola). Se ele acertou uma determinada questão, qual é a probabilidade de ele realmente saber a resposta correta?

Resolução:

$$ P(S \mid A) = \frac{P(S \cap A)}{P(A)} = \frac{0.3}{0.3\cdot1 + 0.70\cdot0.25} \approx 63.16\% $$

In [2]:
def teoremaBayes(pA, pB, pBA):
    return pA * pBA / pB

#  Questão Correta
q = 1/4

#  P(Saber a questão)
pA = 0.30

#  P(Saber e acertar) + P(Acertar no Chute)
pB = (0.30 * 1) + (0.70 * q)

#  P(Acertar e Saber)
#  Se ele sabe, vai acertar a questão.
pBA = 1 

print(f'Resposta: {teoremaBayes(pA, pB, pBA):.2%}') 

Resposta: 63.16%


---

3ª Questão: Urna 1 contém 8 bolas pretas e 2 brancas, urna 2 contém 4 pretas e 5 brancas, e urna 3 contém 7 pretas e 2 brancas. Passa-se uma bola da urna 1 para a urna 2, e depois passa-se uma bola da urna 2 para a urna 3. A seguir retiram-se 2 bolas da urna 3, com reposição. Qual é a probabilidade de que essas 2 bolas sejam brancas?

Resolução:

$$ P(1) = \frac{3}{10} \cdot \frac{3}{10} \cdot 0.6 \cdot 0.2 = 1.08 \% $$
$$ P(2) = \frac{2}{10} \cdot \frac{2}{10} \cdot 0.4 \cdot 0.2 = 0.32 \% $$
$$ P(3) = \frac{3}{10} \cdot \frac{3}{10} \cdot 0.5 \cdot 0.8 = 3.60 \% $$
$$ P(4) =\frac{2}{10} \cdot \frac{2}{10} \cdot 0.5 \cdot 0.8 = 1.60 \% $$

$$ \sum =  6.60 \% $$

In [3]:
from fractions import Fraction

P_1 = (Fraction(3,10) * Fraction(3,10) * Fraction(6,10) * Fraction(2,10))
print(f'{float(P_1):.2%}')
P_2 = (Fraction(2,10) * Fraction(2,10) * Fraction(4,10) * Fraction(2,10))
print(f'{float(P_2):.2%}')
P_3 = (Fraction(3,10) * Fraction(3,10) * Fraction(5,10) * Fraction(8,10))
print(f'{float(P_3):.2%}')
P_4 = (Fraction(2,10) * Fraction(2,10) * Fraction(5,10) * Fraction(8,10))
print(f'{float(P_4):.2%}')

print(f'a) Soma das Probabilidades: {float(P_1 + P_2 + P_3 + P_4):.2%}')

1.08%
0.32%
3.60%
1.60%
a) Soma das Probabilidades: 6.60%


---

4ª Questão: Uma companhia produz circuitos integrados em três fábricas, I, II e III. A fábrica
I produz 40% dos circuitos, enquanto a II e III produzem 30% cada uma. As probabilidades
de que um circuito integrado produzido por estas fábricas não funcione são p, 0; 05 e 0; 04,
respectivamente

a) Determine p de modo que um circuito escolhido aleatoriamente da produção conjunta das
três fábricas tenha probabilidade de não funcionar igual a 3,5%? R: Probabilidade de falha na fabrica 1: 2.00%


b) Dado que o circuito escolhido funciona, qual é a probabilidade de que tenha vindo da
fábrica III (considere p como obtido no item a))?

**Respostas:**

a) Probabilidade de falha na fabrica 1: 2.00%

b) Probabilidade do circuito ter vindo da fabrica 3, dado que funciona: 29.84%

Resolução:

In [4]:
#  4ª Questão: 
#  a)
from sympy import Eq, solve, symbols
from sympy.abc import p

#  Probabilidades do circuito pertencer a fábrica
P_F1 = 0.40
P_F2 = 0.30
P_F3 = 0.30

#  Probabilidade de falha dado a fabrica
P_R_F1 = ''
P_R_F2 = 0.05
P_R_F3 = 0.04

#  Probabilidade de não funcionar
P_R = 0.035

eqn = Eq(p * P_F1 + P_R_F2 * P_F2 + P_R_F3 * P_F3, P_R)
p = solve(eqn, p)
P_R_F1 = p[0]
print(f'a) Probabilidade de falha na fabrica 1: {float(P_R_F1):.2%}')

a) Probabilidade de falha na fabrica 1: 2.00%


In [5]:
#  4ª Questão: 
#  b)

#  P(F3|B) = P(B|F3) * P(F3) / P(B)
#  O problema dá a probabilidade de falha, então, subtraimos 1 para considerar a possibilidade de sucesso.
P_F3_B = (P_F3 * (1-P_R_F3)) / ((P_F1 * (1-P_R_F1)) + (P_F2 * (1-P_R_F2)) + (P_F3 * (1-P_R_F3)))

print(f'b) Probabilidade do circuito ter vindo da fabrica 3, dado que funciona: {float(P_F3_B):.2%}')

b) Probabilidade do circuito ter vindo da fabrica 3, dado que funciona: 29.84%


---

5ª Questão: Uma pessoa tem 2 moedas uma justa e outra falsa, com cara nos dois lados. Ele escolhe uma aleatoriamente.

a) Ele joga a moeda e o resultado é cara; qual é a probabilidade de que ele tenha escolhido a moeda justa?

b) Ele joga novamente a moeda, e novamente sai cara; qual é a probabilidade de que ele tenha escolhido a moeda justa?

c) Ele joga mais uma vez a moeda, e desta vez sai coroa; qual é a probabilidade de que ele tenha escolhido a moeda justa

**Respostas:**

a) Probabilidade de ser a moeda justa, dado que saiu cara: 33.33%

b) Probabilidade de ser a moeda justa, tirando cara duas vezes: 20.00%

c) A moeda só pode ser Justa, dado que a probabilidade da moeda falsa dar cara é igual a Zero.

Resolução:

In [6]:
#  5ª Questão: 
#  a)

#  Probabilidade Moeda Justa
P_MJ = 0.5

#  Probabilidade Moeda Falsa
P_MF = 0.5

#  Probabilidade de dar cara dado Moeda Justa 
P_C_MJ = 0.5

#  Probabilidade de dar cara dado Moeda Falsa 
P_C_MF = 1

#  P(MJ|C)
P_MJ_C = (P_MJ * P_C_MJ) / ((P_MJ * P_C_MJ) + (P_MF * P_C_MF))
print(f'a) Probabilidade de ser a moeda justa, dado que saiu cara: {float(P_MJ_C):.2%}')

a) Probabilidade de ser a moeda justa, dado que saiu cara: 33.33%


In [7]:
#  5ª Questão: 
#  b)

#  Probabilidade Cara - Cara, Moeda Falsa
P_C_C_MF = P_MF * P_C_MF * P_C_MF           # 0.5

#  Probabilidade Cara - Cara, Moeda Justa
P_C_C_MJ = P_MJ * P_C_MJ * P_C_MJ           # 0.125

#  Probabilidade Cara - Coroa, Moeda Justa
P_C_K_MJ = P_MJ * P_C_MJ * (1-P_C_MJ)       # 0.125

#  Probabilidade Coroa - Cara, Moeda Justa
P_K_C_MJ = P_MJ * (1-P_C_MJ) * P_C_MJ       # 0.125

#  Probabilidade Coroa - Coroa, Moeda Justa
P_K_K_MJ = P_MJ * (1-P_C_MJ) * (1-P_C_MJ)   # 0.125

#  Probabilidade Coroa - Coroa, dado Moeda Justa
#  Evento / Possiveis eventos.
P_MJ_C_C = P_C_C_MJ / (P_C_C_MF + P_C_C_MJ)

print(f'b) Probabilidade de ser a moeda justa, tirando cara duas vezes: {float(P_MJ_C_C):.2%}')

b) Probabilidade de ser a moeda justa, tirando cara duas vezes: 20.00%


In [8]:
#  5ª Questão: 
#  c)
print(f'c) A moeda só pode ser Justa, dado que a probabilidade da moeda falsa dar cara é igual a Zero')  

c) A moeda só pode ser Justa, dado que a probabilidade da moeda falsa dar cara é igual a Zero


---

6ª Questão: Uma empresa aérea sabe que 5% das pessoas que fazem reserva não aparecem. Por isto vende 52 assentos para um vôo com 50 lugares. No dia do vôo, qual é a probabilidade de todos os passageiros embarcarem?

**Resposta:**

Probabilidade de 51 ou mais comparecerem: 74.05%

Resolução:

In [9]:
#  6ª Questão:
from scipy.stats import binom

#  Possibilidade de Comparecer
A = 1-0.05

#  Possibilidade de não Comparecer
q = 0.05

#  Numero de vendas
N = 52

#  Numero de lugares disponiveis
K = 50 

#  Usando a formula da Distribuição Cumulativa Binomial. 
P = binom.cdf(K, N, A)

print(f'Probabilidade de 51 ou mais comparecerem: {float(P):.2%}')

from sympy import *
#  Demonstração Matemática

# P_51 = (52*0.95**51*0.05**1)
P_51 = binomial(52,51)*(A**51)*(q**1)

# P_52 = (1*0.95**52*0.05**0)
P_52 = binomial(52,52)*(A**52)*(q**0)

P = 1 - (P_51 + P_52)
print(f'Probabilidade de 51 ou mais comparecerem: {float(P):.2%}')

Probabilidade de 51 ou mais comparecerem: 74.05%
Probabilidade de 51 ou mais comparecerem: 74.05%


---

7ª Questão:  Suponha que uma ação vale inicialmente S(0) = R$ 12; 00, e que a cada instante possa aumentar de valor em 3% com probabilidade p = 0; 45 ou diminuir de valor em 2% com probabilidade 1 − p = 0; 55.

a) Qual é a probabilidade da ação valer mais que seu valor inicial no instante t = 3.

b) Qual é a probabilidade da ação valer mais que R$ 13; 00 no instante t = 15 (se preferir, use a planilha excel para resolver esse item).

**Respostas:**

a) Probabilidade da ação valer mais que 12.0 no instante 3: 42.52%

b) Probabilidade da ação valer mais que 13.0 no instante 15: 34.65%

Resolução:

In [10]:
#  7ª Questão:
#  a)
import numpy as np 
import pandas as pd
from scipy.stats import binom

#  Valor Inicial
VI = 12.0

#  Aumento, probabilidade
A, P_A = 0.03, 0.45

#  Redução, probabilidade
R, P_R = 0.02, 0.55

#  Aumento Aumento Aumento
P_A_A_A = P_A * P_A * P_A
V_A_A_A = VI * (1+A) * (1+A) * (1+A)

#  Aumento Aumento Redução
P_A_A_R = P_A * P_A * P_R
V_A_A_R = VI * (1+A) * (1+A) * (1-R)

#  Aumento Redução Aumento
P_A_R_A = P_A * P_R * P_A
V_A_R_A = VI * (1+A) * (1-R) * (1+A)

#  Aumento Redução Redução
P_A_R_R = P_A * P_R * P_R
V_A_R_R = VI * (1+A) * (1-R) * (1-R)

#  Redução Aumento Aumento
P_R_A_A = P_R * P_A * P_A
V_R_A_A = VI * (1-R) * (1+A) * (1+A)

#  Redução Aumento Redução
P_R_A_R = P_R * P_A * P_R
V_R_A_R = VI * (1-R) * (1+A) * (1-R)

#  Redução Redução Aumento
P_R_R_A = P_R * P_R * P_A
V_R_R_A = VI * (1-R) * (1-R) * (1+A)

#  Redução Redução Redução
P_R_R_R = P_R * P_R * P_R
V_R_R_R = VI * (1-R) * (1-R) * (1-R)

lista_prob = [P_A_A_A, P_A_A_R, P_A_R_A, P_A_R_R, P_R_A_A, P_R_A_R, P_R_A_R, P_R_R_R]
lista_valor = [V_A_A_A, V_A_A_R, V_A_R_A, V_A_R_R, V_R_A_A, V_R_A_R, V_R_A_R, V_R_R_R]

df = pd.DataFrame({'Probabilidade': lista_prob,
                   'Valor Instante 3': lista_valor})

display(df)

p_i_3 = df.loc[df['Valor Instante 3'] >= VI, 'Probabilidade']
p_i_3 = p_i_3.sum()

print(f'a) Probabilidade da ação valer mais que {VI} no instante 3: {p_i_3:.2%}')
#print(sum(lista_prob))
#print(sum(lista_valor))
#lista_valor

Probabilidade  Valor Instante 3
0       0.091125         13.112724
1       0.111375         12.476184
2       0.111375         12.476184
3       0.136125         11.870544
4       0.111375         12.476184
5       0.136125         11.870544
6       0.136125         11.870544
7       0.166375         11.294304

a) Probabilidade da ação valer mais que 12.0 no instante 3: 42.52%


In [11]:
#  7ª Questão:
#  b)
import numpy as np 
import pandas as pd
from scipy.stats import binom

#  Taxa de Aumento
Tx_A = 1.03

#  Taxa de Redução
Tx_R = 0.98

#  Instante Final
IF = 15 

#  Criando Data Frame
t = np.arange(0,IF+1)
df = pd.DataFrame({'Instante': t})

df['Preço'] = round(VI * (Tx_A**df['Instante']) * (Tx_R**(IF-df['Instante'])), 2)
df['Probabilidade'] = binom.pmf(df['Instante'], IF, P_A)
df['Probabilidade Acum'] = binom.cdf(df['Instante'], IF, P_A)
display(df)

#  Probabilidade de valer mais que 13.0 reais:
v_13 = df.loc[df['Preço'] >= 13, 'Probabilidade']
v_13 = v_13.sum()

print(f'b) Probabilidade da ação valer mais que 13.0 no instante 15: {v_13:.2%}')

Instante  Preço  Probabilidade  Probabilidade Acum
0          0   8.86       0.000127            0.000127
1          1   9.32       0.001565            0.001692
2          2   9.79       0.008960            0.010652
3          3  10.29       0.031769            0.042421
4          4  10.81       0.077978            0.120399
5          5  11.37       0.140360            0.260760
6          6  11.95       0.191401            0.452160
7          7  12.56       0.201344            0.653504
8          8  13.20       0.164736            0.818240
9          9  13.87       0.104832            0.923071
10        10  14.58       0.051463            0.974534
11        11  15.32       0.019139            0.993673
12        12  16.10       0.005220            0.998893
13        13  16.92       0.000986            0.999879
14        14  17.79       0.000115            0.999994
15        15  18.70       0.000006            1.000000

b) Probabilidade da ação valer mais que 13.0 no instante 15: 34.65%


---

8ª Questão:  Um lote de 50 peças é enviado para inspeção. O inspetor testa 5 peças e aceita o lote diretamente se as 5 peças forem consideradas boas. Se pelo menos uma for defeituosa todas as 50 peças são testadas. Considerando um lote com 6 peças defeituosas, determine a probabilidade do lote ser aceito diretamente.

**Resposta:**

Probabilidade do lote ser aceito diretamente: 38.44%

Resolução:

In [12]:
#  8ª Questão:
import numpy as np 
import pandas as pd
from scipy.stats import hypergeom

#  Quantidade para não ser aceito
k = 1 

#  Numero peças
M = 50

#  Tamanho da Amostra
n = 5

#  Peças Defeituosas
N = 6

#  Probabilidade Hypergeometrica pmf(k, M, n, N)
prb = hypergeom.pmf(k, M, n, N)

print(f'Probabilidade do lote ser aceito diretamente: {prb:.2%}')

Probabilidade do lote ser aceito diretamente: 38.44%


---

9ª Questão:  Uma fábrica de carros sabe que os motores de sua fabricação têm duração de acordo com uma distribuição normal com média µ = 150:000 Km e desvio-padrão σ = 5:000 km. Qual é a probabilidade de que um carro qualquer tenha um motor que dure:

a) Pelo menos 160.000 Km?

b) Entre 140.000 e 165.000 Km?

c) Se a fábrica substitui o motor que apresente duração inferior à garantia, qual deve ser a garantia em Km para que a porcentagem de motores substituídos seja inferior à 0,2%?

**Respostas:**

a) Pelo menos 160000km: 2.28%

b) Entre 140.000 e 165.000 km: 97.59%

c) A garantia deve ser de: 139731.26 km

Resolução:

In [13]:
#  9ª Questão:
#  a)
import numpy as np 
import pandas as pd
from scipy.stats import norm

#  Média
media = 150000 

#  Desvio
std_dev = 5000

#  Distribuição normal.
prb = (1 - norm.cdf(160000, media, std_dev))

print(f'a) Pelo menos 160.000 km: {prb:.2%}')

a) Pelo menos 160.000 km: 2.28%


In [14]:
#  9ª Questão:
#  b)
import numpy as np 
import pandas as pd
from scipy.stats import norm

#  Média
media = 150000 

#  Desvio
std_dev = 5000

#  Distribuição normal.
prb_1 = (1 - norm.cdf(140000, media, std_dev))
prb_2 = (1 - norm.cdf(165000, media, std_dev))

# P(140000 < Z < 165000)
prb = (prb_1 - prb_2)

print(f'b) Entre 140.000 e 165.000 km: {prb:.2%}')

b) Entre 140.000 e 165.000 km: 97.59%


In [15]:
#  9ª Questão:
#  c)
import numpy as np 
import pandas as pd
from scipy.stats import norm

#  Probabilidade dada no enunciado.
probabilidade = 0.02

#  Média
media = 150000 

#  Desvio
std_dev = 5000

#  Gausseana Inversa 
prb = (norm.ppf(probabilidade, media, std_dev))

print(f'c) A garantia deve ser de: {prb:.2f} km')

c) A garantia deve ser de: 139731.26 km


---

10ª Questão:  Uma pessoa decide investir em um título cambial comprando US$ 200000,00. A taxa de câmbio atual real/dolar é 2,0. Suponha que o desvio padrão diário da taxa de câmbio R seja 5,0% e que possa ser modelada por uma variável aleatória normal. Qual é a perda máxima em real em 1 dia com 95% de chances?

**Resposta:**

Perda maxima em 1 dia: 32897.07 de reais

Resolução:

In [16]:
#  10ª Questão:
import numpy as np 
import pandas as pd
from scipy.stats import norm

#  Taxa Cambio
tx_camb = 2 

#  Investimento Inicial
invest_ini = 200000 * tx_camb 

#  Desvio padrão
std_dev = 0.05 * invest_ini

#  Probabilidade
probabilidade = 0.95

#  Distribuição normal.
distr = (norm.ppf(probabilidade, invest_ini, std_dev))
var = distr - invest_ini

print(f'Perda maxima em 1 dia: {var:.2f} de reais')

Perda maxima em 1 dia: 32897.07 de reais


---

11ª Questão:  Considere um jogo em que se paga 25,00 para entrar, e se joga 3 dados. Se apenas uma das faces der 1 ganha-se 25,00, se exatamente 2 faces derem 1 ganha-se 225,00, e se as 3 faces derem 1 ganha-se R$425,00. Qualquer outro resultado não se ganha nada. Calcule: 

i) o lucro esperado do jogador e responda se vale a pena ou não entrar nesse jogo; 

ii) a variância do lucro do jogador

**Respostas:**

i) Lucro esperado: 1.27 de reais

ii) Variancia do jogador: 3878.59

Resolução:

In [17]:
#  11ª Questão:
#  I)
prob_0 = (5/6)*(5/6)*(5/6)
lucro_0 = 0 - 25

prob_1 = 3*(1/6)*(5/6)*(5/6)
lucro_1 = 25 - 25

prob_2 = 3*(1/6)*(1/6)*(5/6)
lucro_2 = 225 - 25

prob_3 = (1/6)*(1/6)*(1/6)
lucro_3 = 425 - 25

prob = [prob_0, prob_1, prob_2, prob_3]
lucro = [lucro_0, lucro_1, lucro_2, lucro_3]

lucro_esperado = sum(x * y for x, y in zip(lucro, prob))
print(f'i) Lucro esperado: {lucro_esperado:.2f} de reais')

i) Lucro esperado: 1.27 de reais


In [18]:
#  11ª Questão:
#  II)

variancia = sum(y*((x - lucro_esperado)**2) for x, y in zip(lucro, prob))
print(f'ii) Variancia do jogador: {variancia:.2f}')

ii) Variancia do jogador: 3878.59


---

12ª Questão:  O verão na cidade de São Sebastião é classificado como sendo um verão chuvoso ou um verão ensolarado. Os lucros das 2 principais empresas de São Sebastião, o hotel São Sebastião e a indústria de guarda-chuvas São Sebastião dependem da classificação do verão, conforme a tabela 2. Sabe-se que 30% dos verões são chuvosos e 70% são ensolarados. Sejam as variáveis X1 e X2 o lucro do hotel São Sebastião e da indústria de guarda-chuvas São Sebastião respectivamente. Pergunta-se: 

a) Qual é o fator de correlação entre X1 e X2? 

b) Suponha que voce seja dono de a% do hotel e (100 − a)% da indústria de guarda-chuvas, com 0 ≤ a ≤ 100. Qual deveria ser o valor de a para se ter risco zero, e qual seria o lucro garantido nesse caso?

**Respostas:**

a) Ro: -1.00 

b) a: 0.57 e o lucro garantido: 1285.71 reais

Resolução:

In [19]:
#  12ª Questão:
#  a)

import numpy as np 

#  Verão Chuvoso
p_verao_chuva = 0.3
lucro_hotel_chuva = -1500
lucro_indus_chuva = 5000

#  Verão Sol
p_verao_sol = 0.7
lucro_hotel_sol = 3000
lucro_indus_sol = -1000


E_x1 = (p_verao_chuva * lucro_hotel_chuva) + (p_verao_sol * lucro_hotel_sol)
E_x2 = (p_verao_chuva * lucro_indus_chuva) + (p_verao_sol * lucro_indus_sol)

E_x1_x2 = (lucro_hotel_chuva*lucro_indus_chuva*p_verao_chuva) + (lucro_hotel_sol*lucro_indus_sol*p_verao_sol)

#  Encontrando a Covariancia
cov_x1_x2 = (E_x1_x2 - (E_x1*E_x2))

#  Encontrando os Sigma
sigma_1 = np.sqrt(p_verao_chuva*(lucro_hotel_chuva-E_x1)**2 + p_verao_sol*(lucro_hotel_sol-E_x1)**2)
sigma_2 = np.sqrt(p_verao_chuva*(lucro_indus_chuva-E_x2)**2 + p_verao_sol*(lucro_indus_sol-E_x2)**2)

#  Encontrando o Ro
ro = cov_x1_x2 / (sigma_1 * sigma_2)

print(f'a) Ro: {ro:.2f}')

a) Ro: -1.00


In [20]:
#  12ª Questão:
#  b)


#  Zerando risco: a = σ2/(σ1 + σ2)
a = sigma_2 / (sigma_1 + sigma_2)

#  Encontrando lucro garnatido
lucro_g = (a * E_x1) + ((1 - a) * E_x2)

print(f'b) a: {a:.2f} e lucro garantido: {lucro_g:.2f} reais')

b) a: 0.57 e lucro garantido: 1285.71 reais
